torch/autograd/functional.py

```diff
@@ -323,6 +323,13 @@ def jvp(func, inputs, v=None, create_graph=False, strict=False):
             jvp (tuple of Tensors or Tensor): result of the dot product with
             the same shape as the output.
 
+    Note:
+        ``autograd.functional.jvp`` computes the jvp by using the backward of
+        the backward (sometimes called the double backwards trick). This is not
+        the most performant way of computing the jvp. Please consider using
+        `functorch's jvp <https://github.com/pytorch/functorch#jvp>`_
+        or the low-level forward-mode AD API instead.
```

Comment:
You can link to https://pytorch.org/docs/master/autograd.html#forward-mode-automatic-differentiation here

Reply:
Yup, it took me a while to figure out how to actually get that cross-reference to work.